src/pages/home/report/ReportActionsList.js

```diff
@@ -175,7 +175,7 @@ class ReportActionsList extends React.Component {
                         // Make sure the oldest report action loaded is not the first. This is so we do not show the
                         // skeleton view above the created action in a newly generated optimistic chat or one with not
                         // that many comments.
-                        const lastReportAction = _.last(this.props.sortedReportActions);
+                        const lastReportAction = _.last(this.props.sortedReportActions) || {};
```

Comment:
This change is fixing a different bug I discovered when working on this PR. When the workspace has been deleted, navigating to the archived chats threw error, because the `this.props.sortedReportActions` was an empty string resulting in `lastReportAction` being `undefined`. Hence next line threw error.